lib/ext/heroku/command/addons.rb

```diff
@@ -308,15 +313,13 @@ def detach
           :method   => :delete,
           :path     => "/addon-attachments/#{addon_attachment['id']}"
         ).body
-      end
-      action("Unsetting #{attachment_name}_URL and restarting #{app}") do
```

Comment:
Definitely a good call here. Whatever happens to the L259 change, there should be some inverse output here, though. Again, maybe either checking that the release number has actually changed or including vars in the serializer is the way to go here.